webapp/src/main/java/com/box/l10n/mojito/common/notification/SlackWarningMessageSender.java

```diff
@@ -0,0 +1,80 @@
+package com.box.l10n.mojito.common.notification;
+
+import com.box.l10n.mojito.entity.TMTextUnit;
+import com.box.l10n.mojito.service.assetintegritychecker.integritychecker.HtmlTagIntegrityCheckerException;
+import com.box.l10n.mojito.service.assetintegritychecker.integritychecker.IntegrityCheckException;
+import com.box.l10n.mojito.slack.SlackClient;
+import com.box.l10n.mojito.slack.SlackClientException;
+import com.box.l10n.mojito.slack.SlackClients;
+import com.box.l10n.mojito.slack.request.Message;
+import jakarta.annotation.PostConstruct;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * Sends warning messages to the specified warning channel on Slack. Selects default Slack client
+ * specified in app properties, if none specified the first Slack Client is used. The channel name
+ * for warnings must be placed in the app properties under: l10n.slackWarningChannel
+ *
+ * @author mattwilshire
+ */
+@Component
```

Comment:
Should probably make the presence of this component configurable, see `@ConditionalOnProperty`